### PR TITLE
Creating the current copy of the DEV to EA

### PR DESCRIPTION
**Pull request by SnowNooDLe**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+.env
+.env.dev
+.env.test
+
+# Pthon virtual environment
+/venv
+
+# vscode setting file
+/.vscode
```
Comment (jasonmotha) [inline .gitignore:20]: Some of these can be excluded from your .git/exclude folder rather than the .gitignore file

**Review by jasonmotha — CHANGES_REQUESTED** [review]
Is there any way to autogenerate the yarn / package-lock files? It seems bulky to have 20k lines like this in git.

File: .gitignore (hunk 1 above)
Comment (claudio525) [reply .gitignore:20]: The `exclude` file in `.git/info` folder, is a private .gitignore (i.e. doesn't get commited)

**Review by claudio525 — COMMENTED** [review]
Agree with @jasonmotha, don't think `package-lock.json` needs to be commited. From my understanding that is generated when one runs `npm install`, which only needs `package.json`?

**Review by jasonmotha — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Pull request merged**